src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderAddressRepository.java

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.hmpps.prison.repository.jpa.repository;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+import uk.gov.justice.hmpps.prison.repository.jpa.model.OffenderAddress;
+
+@Repository
+public interface OffenderAddressRepository extends CrudRepository<OffenderAddress, Long> {
```

Comment:
Not used yet - I assume it is going to be?

Reply:
correct its all work in progress